content/src/content/jcr_root/apps/core/wcm/components/modal/v1/modal/modal.html

```diff
@@ -0,0 +1,32 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2019 Adobe Systems Incorporated
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+
+<div data-sly-use.modal="com.adobe.cq.wcm.core.components.models.Modal"
+     data-sly-use.templates="core/wcm/components/commons/v1/templates.html"
+     role="dialog">
+    <sly data-sly-test="${wcmmode.edit}">${modal.title || 'Dialog' @ i18n} - ${modal.id}</sly>
+    <sly data-sly-test="${!wcmmode.edit}">
+        <sly data-sly-test.xfPathWithExt="${modal.experienceFragmentPath @ extension='html'}"></sly>
+        <div class="cmp-modal"
+             id="${modal.id}"
+             data-modal-show="${properties.open}"
+             tabindex="-1"
+             role="dialog"
+             data-content-url="${properties.fragmentType == 'cf' ? modal.contentFragmentPath : xfPathWithExt}">

```

Comment:
@gitrajat Currently CF doesn't have any extension. It will work fine in author and publisher but will mostly get blocked by dispatcher filter.
@vladbailescu Which extension should be used for CF content retrieval which would be allowed by the dispatcher? .cfm.content.html ?

Reply:
@vladbailescu Any inputs?